source/AI.cpp

```diff
@@ -1117,6 +1117,10 @@ shared_ptr<Ship> AI::FindTarget(const Ship &ship) const
 	if(oldTarget && person.IsTimid() && oldTarget->IsDisabled()
 			&& ship.Position().Distance(oldTarget->Position()) > 1000.)
 		oldTarget.reset();
+	// Ships with 'plunders' personality always destroy the ships they have boarded.
+	if(oldTarget && oldTarget->IsDisabled() && person.Plunders()
+			&& !person.Disables() && Has(ship, oldTarget, ShipEvent::BOARD))
```

Comment:
Check personalities before checking the status of the target as is done for the timid check.
```suggestion
	if(oldTarget && person.Plunders() && !person.Disables() 
			&& oldTarget->IsDisabled() && Has(ship, oldTarget, ShipEvent::BOARD))
```